### PR TITLE
added confluence dataloader

### PR DESCRIPTION
**Pull request by zywilliamli**
adds a data connector for confluence, given a space id and/or a list of page ids

### CODE REVIEW

File: loader_hub/confluence/base.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            api_token = os.getenv(CONFLUENCE_API_TOKEN)
+            if api_token is None:
+                raise ValueError(
+                    "Must specify `api_token` or set environment "
+                    "variable `CONFLUENCE_API_TOKEN`."
+                )
+
+        self.confluence = Confluence(url=base_url, username=user_name, password=api_token, cloud=True)
+
+    def load_data(self, space_id: Optional[str] = None, page_ids: Optional[List[str]] = []) -> List[Document]:
```
Comment (jerryjliu) [inline loader_hub/confluence/base.py:48]: nit: I'd set default to `page_ids` to None, and do `page_ids = page_ids or []` within the function. There's some dangers to initializing an empty list in the function signature

File: loader_hub/confluence/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Confluence Loader
+
+This loader loads pages from a given Confluence site/instance. The user specifies a username, API token and the base URL for a Confluence instance to initialize
+the ConfluenceReader. They then specify a set of `page_ids` and/or `space_id` to load in
+the corresponding Document objects, if both are specified the union of both sets will be returned. 
+
+## Usage
+
+Here's an example usage of the NotionPageReader.
```
Comment (jerryjliu) [inline loader_hub/confluence/README.md:9]: nit: ConfluenceReader

**Review by jerryjliu — COMMENTED** [review]
amazing! a few minor comments

File: loader_hub/confluence/README.md (hunk 2 above)
Comment (zywilliamli) [reply loader_hub/confluence/README.md:9]: 🤦 thx

**Review by jerryjliu — APPROVED** [review]

**Pull request merged**